{{cookiecutter.project_slug}}/ios/fastlane/Fastfile

```diff
@@ -139,32 +148,32 @@ platform :ios do
       }
     }
     headers = {
-      "Authorization": "Api-Key #{ENV['WEBHOOK_API_KEY']}",
-      "Content-Type": "application/json"
+      'Authorization': "Api-Key #{ENV['WEBHOOK_API_KEY']}",
+      'Content-Type': 'application/json'
     }
+
     response = HTTParty.post(url, body: data.to_json, headers: headers)
-    puts "API response: #{response.code} #{response.body}"
+    puts("API response: #{response.code} #{response.body}")
 
     response.success?
   end
 end
 
-# get the publicKey of the appetizeApp corresponding to your base_name
 def get_appetize_public_key(platform, base_name)
-  return ENV['APPETIZE_PUBLIC_KEY'] unless ENV['APPETIZE_PUBLIC_KEY'].nil?
+  @appetize_key ||= ENV['APPETIZE_PUBLIC_KEY']
+  @appetize_key ||= fetch_appetize_public_key(platform, base_name)
+  @appetize_key
```

Comment:
This line with `@appetize_key` isn't necessary. The previous line will already return `@appetize_key`

Reply:
added for explicitness